Assets/_Core/Event/ExoEvent/Scripts/CharacterBehaviour.cs

```diff
@@ -3,16 +3,16 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-[RequireComponent(typeof(Rigidbody)), RequireComponent(typeof(Collider))]
+[RequireComponent(typeof(Rigidbody), typeof(Collider))]
 public class CharacterBehaviour : MonoBehaviour
 {
-    private int _health = 100;
+    private int _healthInt = 100;
```

Comment:
Le nom que tu avais avant été bien. Là je pense que la précision est peut-être un peu trop.
Ou alors, si vraiment tu veux lever une ambigüité, je verrais bien plutôt `_healthValue`.